src/app/cube/details/action-panel/action-panel.component.html

```diff
@@ -19,6 +19,11 @@
         <i class="fal fa-spinner-third fa-spin"></i>
       </span>
     </button>
+
+    <button (click)="clark.center" class="button neutral on-white" id="edit-button" type="submit"> 
+      {{ 'Revise' }}
```

Comment:
Since this is not making a function call or computing any values, we could actually just directly write the text into the HTML instead of having Angular render it:
```suggestion
      Revise
```